packages/logo/composer.json

```diff
@@ -4,6 +4,7 @@
 	"type": "library",
 	"license": "GPL-2.0-or-later",
 	"require": {},
+	"version": "1.0.0",
```

Comment:
Let's remove this as well, once we change the stability constraint in the main composer.json to `@dev`.